xml/System.Windows.Forms/ComboBox.xml

```diff
@@ -3028,11 +3028,21 @@
         <Parameter Name="keyData" Type="System.Windows.Forms.Keys" Index="1" FrameworkAlternate="netcore-3.0;netcore-3.1" />
       </Parameters>
       <Docs>
-        <param name="msg">To be added.</param>
-        <param name="keyData">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="msg">A <see cref="T:System.Windows.Forms.Message" />, passed by reference, that represents the window message to process.</param>
+        <param name="keyData">One of the <see cref="T:System.Windows.Forms.Keys" /> values that represents the key to process.</param>
```

Comment:
`System.Windows.Forms.Keys` is an enum, so the preferred way to describe this parameter would be something like this:

```suggestion
        <param name="keyData">One of the enumeration values that represent the key to process.</param>
```

Reply:
Below, in line 14886 of DataGridView.xml, your description of the parameter says it's a bitwise combination of this enum's values. Should this parameter also be described as a bitwise combination, or does it only allow one value?